File: backend/src/ai/objects/apiTypes/AIMerchantProductDescriptionsResponse.scala
package delivery.ai.objects.apiTypes

import delivery.ai.objects.AIGeneratedProductDescription
import delivery.shared.objects.MerchantId

final case class AIMerchantProductDescriptionsResponse(
    merchantId: MerchantId,
    products: List[AIGeneratedProductDescription],
    generatedAt: String
)

File: frontend/src/objects/ai/apiTypes/AIMerchantProductDescriptionsResponse.ts
import type { MerchantId } from '@/objects/shared/ids'
import type { AIGeneratedProductDescription } from '../AIGeneratedProductDescription'

export interface AIMerchantProductDescriptionsResponse {
  merchantId: MerchantId
  products: AIGeneratedProductDescription[]
  generatedAt: string
}
